apps/designer/app/designer/features/style-panel/shared/css-value-input/keyword-utils.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * Beautify a CSS keyword by capitalizing the first letter of each word,
+ * and replacing dashes with spaces.
+ */
+export const toPascalCaseNoDashes = (keyword: string) => {
```

Comment:
```suggestion
export const toPascalCaseWithoutDashes = (keyword: string) => {
```

Reply:
but I would just use toPascalCase, since its a specialized utility for keywords

Reply:
additionally we could have a cache, because its a limited set of keywords, it will be all cached at one point